src/mus2midi.c

```diff
@@ -0,0 +1,223 @@
+/*
+ * rv32emu is freely redistributable under the MIT License. See the file
+ * "LICENSE" for information on usage and redistribution of this file.
+ */
+
+/*
+ * Reference: https://github.com/fawtytoo/Mus2Midi
```

Comment:
Remove the hyperlink of Mus2Midi and put the license notice below:
```c
/* Copyright (C) 2021-2022 Steve Clark
 * 
 * This software is provided 'as-is', without any express or implied
 * warranty.  In no event will the authors be held liable for any damages
 * arising from the use of this software.
 *
 * Permission is granted to anyone to use this software for any purpose,
 * including commercial applications, and to alter it and redistribute it
 * freely, subject to the following restrictions:
 *
 * 1. The origin of this software must not be misrepresented; you must not
 *  claim that you wrote the original software. If you use this software
 *  in a product, an acknowledgment in the product documentation would 
 *  be
 *  appreciated but is not required. 
 * 2. Altered source versions must be plainly marked as such, and must not be
 *  misrepresented as being the original software.
 * 3. This notice may not be removed or altered from any source distribution.
*/
```

Reply:
See https://github.com/fawtytoo/Mus2Midi/blob/main/Licence

Reply:
`rv32emu` is licensed under an MIT-style license, which is compatible with the zlib license used by the `MusMidi` project. However, since the license specifies that "This notice may not be removed or altered from any source distribution," we are obligated to comply.